software/workspace/workspace.go

```diff
@@ -102,11 +116,57 @@ func GetCurrentWorkspace() (string, error) {
 	return c.Workspace, nil
 }
 
-func getWorkspaceSelection(client houston.ClientInterface, out io.Writer) (string, error) {
+// WorkspacesPromptPaginatedOption Show pagination option based on page size and total record
+var WorkspacesPromptPaginatedOption = func(pageSize, pageNumber, totalRecord int) PaginationOptions {
```

Comment:
```suggestion
var workspacesPromptPaginatedOption = func(pageSize, pageNumber, totalRecord int) PaginationOptions {
```

function doesn't needs to be public, since it is only referred inside this package